mkosi/__init__.py

```diff
@@ -2287,6 +2309,10 @@ def target() -> None:
     if build and args.auto_bump:
         bump_image_version()
 
+    # Give disk images some space to play around with if we're booting one.
+    if args.verb in (Verb.shell, Verb.boot, Verb.qemu) and last.output_format == OutputFormat.disk:
+        run(["truncate", "-s", "8G", last.output_dir / last.output])
```

Comment:
```suggestion
        run(["truncate", "--size", "8G", last.output_dir / last.output])
```
This does make the minimum size of the image file 8G when we build it in the context of either of these verbs, right? That should probably be mentioned in the news lest we forget.